src/Service/ActivityPubManager.php

```diff
@@ -337,6 +341,10 @@ public function updateUser(string $actorUrl): ?User
             return $user;
         }
 
+        if ($this->settingsManager->isBannedInstance($actorUrl)) {
+            return null;
+        }
```

Comment:
This check is be redundant as this updateActor and createUser which both have the check already

Reply:
Nope, again.. As stated in my PR:

> There are several places like LikeHandler, UpdateHandler and UpdateActorHandler that is calling methods directly or indirectly methods from ActivityPubManager.php.

Despite you adding checks present in this file, this public method is also called outside of AcitivtyPubManager.php namely in the handlers.

Reply:
Agreed. Removing this again. And marking this method as private.